yarn.lock

```diff
@@ -0,0 +1,331 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I think so. Having the package.json should be enough for the installing the dependencies right?

Reply:
Actually we do need a yarn.lock, but one already exists in the `/frontend` folder of the repo since the depedencies are all for frontend. This one actually shouldn't be doing anything regardless as calling `yarn` or `yarn install` here would just create a new `/node_modules` folder at the root rather than in the frontend folder.

Reply:
LOL that explains issues I was having with a random node_modules folder popping up in the root directory